README.rst

```diff
@@ -35,32 +44,54 @@ Next, configure your aws credentials and default region:
     Default region name [us-east-1]:
     Default output format [None]:
 
-Then, run pcluster configure:
+Then, run ``pcluster configure``. A list of valid options will be displayed for each
+configuration parameter: type an option number and press ``Enter`` to confirm your choice, or just press ``Enter``
+to accept the default option.
 
 .. code-block:: ini
 
   $ pcluster configure
-  Cluster Template [default]:
-  Acceptable Values for AWS Region ID:
-      ap-south-1
-      ...
-      us-west-2
+  INFO: Configuration file /dir/conf_file will be written.
+  Press CTRL-C to interrupt the procedure.
+
+
+  Allowed values for AWS Region ID:
+  1. eu-north-1
+  ...
+  15. us-west-1
+  16. us-west-2
   AWS Region ID [us-east-1]:
-  VPC Name [myvpc]:
-  Acceptable Values for Key Name:
-    keypair1
-    keypair-test
-    production-key
-  Key Name []:
-  Acceptable Values for VPC ID:
-    vpc-1kd24879
-    vpc-blk4982d
-  VPC ID []:
-  Acceptable Values for Master Subnet ID:
-    subnet-9k284a6f
-    subnet-1k01g357
-    subnet-b921nv04
-  Master Subnet ID []:
+  ...
+
+Make sure to select a region on which you own at least an EC2 key pair.
+
+During the process you will be asked to setup your networking environment. The wizard will
+offer you the possibility of creating a new VPC on the fly.
+
+.. code-block:: ini
+
+  Automate VPC creation? (y/n) [n]:
+
+Enter '``n``' if you already have a correctly set up VPC. Otherwise you can let ``pcluster configure``
```

Comment:
"correctly set up" is vague. Does it mean that a customer has already specified in the config file the VPC and subnet ID that they want to use? If so, I'd say something like "...if you have already specified the VPC configuration that you wish to use. Otherwise..."

Reply:
What I meant here is: "If you already have a VPC on the AWS cloud that you are sure you can install the cluster on". For instance, I have my VPC with two subnets and I usually create all my clusters there. Could you suggest me a better way to express this concept?

Reply:
Okay, I think that sounds good. "...if you already have a VPC suitable for the cluster."

Reply:
:+1: